src/components/accounts/AccountTransactionsScreen.tsx

```diff
@@ -0,0 +1,384 @@
+  >
+    {props.children}
+  </TouchableHighlight>;
+
+  return (
+    <AccountsContext.Consumer>
+      {({currentAccount: account}) => (
+        <View style={styles.container}>
+          <BackgroundGradient orientation={'horizontal'}>
+            <View style={styles.header}>
+              <Sats sats={account?.summary?.balanceSats} satsStyle={styles.sats} satsLabelStyle={styles.satsLabel} usdStyle={styles.usd} usdLabelStyle={styles.usdLabel} />
+            </View>
+            <GradientSeparator />
+            <View style={styles.actionBar}>
+              <ActionButton style={{
+                  borderRightWidth: 1,
+                  borderRightColor: Colors.grey48,                
+                }}
+                onPress={notImplementedAlert}
+              >
+                <AppText style={styles.actionLabel}>Sign & Send</AppText>
+              </ActionButton>
+              <ActionButton
+                style={{width: '20%'}}
+                onPress={notImplementedAlert}
+              >
+                <CameraIcon width={18} height={13} />
+              </ActionButton>
+              <ActionButton style={{
+                  borderLeftWidth: 1,
+                  borderLeftColor: Colors.grey48,                
+                }}
+                onPress={notImplementedAlert}
+              >
+                <AppText style={styles.actionLabel}>New Invoice</AppText>
+              </ActionButton>
+            </View>
+            <GradientSeparator />
+            <View style={styles.tabs}>
+              <View style={styles.metrics}>
+                <View style={styles.metricContainer}>
+                  <AppText style={[styles.metric, styles.metricSelected]}>{numFormat(account?.summary?.numTransactions)}</AppText>
+                  <View>
+                    <AppText style={[styles.metricLabel, styles.metricLabelSelected]}>Total{"\n"}Transactions</AppText>
+                  </View>
+                </View>
+                <View style={styles.metricContainer}>
+                  <AppText style={styles.metric}>{numFormat(account?.summary?.numAddresses)}</AppText>
+                  <View>
+                    <AppText style={styles.metricLabel}>Child{"\n"}Accounts</AppText>
+                  </View>
+                </View>
+                <View style={styles.metricContainer}>
+                  <AppText style={styles.metric}>{numFormat(account?.summary?.numUtxos)}</AppText>
+                  <View>
+                    <AppText style={styles.metricLabel}>Spendable{"\n"}Outputs</AppText>
+                  </View>
+                </View>
+                <View style={styles.metricContainer}>
+                  <AppText style={styles.metric}>{numFormat(account?.summary?.satsInMempool)}</AppText>
+                  <View>
+                    <AppText style={styles.metricLabel}>Sats in{"\n"}Mempool</AppText>
+                  </View>
+                </View>
+              </View>
+            </View>
```

Comment:
this can be a separate component // ActionTabs

Reply:
same for remaining JXS below, I think this whole file can be divided into 3-4 child component sections. 
It will be easier to read the code and make changes.

Reply:
Good suggestions 👍   I've broken it up into several components.